odm2api/ODM2/services/readService.py

```diff
@@ -694,13 +696,70 @@ def getDataSets(self, codes=None, uuids=None):
         if codes:
             q = q.filter(DataSets.DataSetCode.in_(codes))
         if uuids:
-            q.q.filter(DataSets.DataSetUUID.in_(uuids))
+            q.filter(DataSets.DataSetUUID.in_(uuids))
         try:
             return q.all()
         except Exception as e:
             print('Error running Query {}'.format(e))
             return None
 
+    def getSamplingFeatureDatasets(self, ids=None, codes=None, uuids=None, type=None):
+        """Retrieve a list of Sampling Feature objects.
+        Retrieve a list of Datasets associated with the given sampling feature data.
+
+        Must specify either samplingFeatureID OR samplingFeatureUUID OR samplingFeatureCode)
```

Comment:
Thanks for following the convention that we established when you were gone. 😄 Maybe this line should be bold, so it's not easy to miss?

Reply:
@lsetiawan  do I do that by adding the three > in front of it? and the whole paragraph or just the line that indicates you much specify one of the three  features?

Reply:
I think just the line that indicates that you must specify one of three features. I think you can just do

```
**Must specify either samplingFeatureID OR samplingFeatureUUID OR samplingFeatureCode)**
```

Reply:
@lsetiawan okay, that has been updated. The formatting in the function documentation is just markdown right?

Reply:
It is reStructuredText.